sdk/schemaregistry/test-resources.json

```diff
@@ -138,11 +312,101 @@
         "schemaType": "custom"
       }
     },
+    {
+      "type": "Microsoft.EventHub/namespaces/schemagroups",
```

Comment:
What do you think of using the [ARM library](https://www.nuget.org/packages/Azure.ResourceManager.EventHubs) within the live test class for the schema group creation/deletion rather than test-resources? I'm thinking it will be cumbersome to keep updating this template, the TestEnvironment file, _and_ the live test class whenever a new service version is introduced.

This would add a bit of overhead for running the live tests locally, but given the 25 schema group limit, it is not currently working well anyway so this should still be a net improvement.